test/newsimplification/SimplifyProperties.spec.ts

```diff
@@ -44,30 +46,12 @@ describe('Simplification of properties', () => {
     simplifyProperties(schema, model, simplifier);
     expect(model.addTypes).toHaveBeenNthCalledWith(1, 'object');
   });
-  test('should use as is', () => {
-    const schema: any = { properties: { 'property1': { type: 'string' } } };
-    const model = new CommonModel();
-    const simplifier = new Simplifier();
-    simplifyProperties(schema, model, simplifier);
-    expect(simplifier.simplify).toHaveBeenNthCalledWith(1, { type: 'string' });
-    expect(model).toMatchObject(
-      {
-        properties: { 'property1': {} },
-      },
-    );
-  });
-  test('should merge existing properties', () => {
+  test('should go trough properties and add it to model', () => {
     const schema: any = { properties: { 'property1': { type: 'string' } } };
     const model = new CommonModel();
-    model.properties = { 'property1': new CommonModel() };
     const simplifier = new Simplifier();
     simplifyProperties(schema, model, simplifier);
     expect(simplifier.simplify).toHaveBeenNthCalledWith(1, { type: 'string' });
-    expect(CommonModel.mergeCommonModels).toHaveBeenNthCalledWith(1, expect.anything(), expect.anything(), schema);
-    expect(model).toMatchObject(
-      {
-        properties: { 'property1': {} },
-      },
-    );
+    expect(model.addProperty).toHaveBeenNthCalledWith(1, "property1", mockedSimplifierModel, schema);
   });
 });
```

Comment:
I do not know why but there is some reason to add last line everywhere. You can add eslint rule for that.

Reply:
True, it should be consistent as well, wanna create an issue, or should I? That we should apply https://eslint.org/docs/rules/eol-last

Reply:
Do you by yourself, I have like 3 feature requsts to write for you 😄

Reply:
Awesome 😄 

I will create it.

Reply:
https://github.com/asyncapi/generator-model-sdk/issues/190